MAUI/ProgressBar/getting-started.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: Getting started with .NET MAUI ProgressBar control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI ProgressBar (Progress Bar) control, its elements and more.
+platform: MAUI
+control: ProgressBar
+documentation: ug
+---
+
+# Getting started with MAUI ProgressBar (Progress Bar)
```

Comment:
Getting started with .NET MAUI ProgressBar (Progress Bar)

Reply:
Added .NET MAUI in getting started